pkg/ui/distccl/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+genrule(
+    name = "assets",
+    srcs = ["//pkg/ui/workspaces/db-console:db_console_webpack"],
+    outs = ["assets"],
+    cmd = "cp -r $(locations //pkg/ui/workspaces/db-console:db_console_webpack)/* $(OUTS)",
```

Comment:
Nit: `-R` is more standard than `-r` here.

Reply:
Done